Resources/config/odm.xml

```diff
@@ -87,5 +91,17 @@
             <tag name="validator.constraint_validator" alias="doctrine_phpcr.odm.validator.valid_phpcr_odm"/>
         </service>
 
+        <!-- Translation strategies -->
+        <service id="doctrine_phpcr.odm.translation.strategy.attribute" class="%doctrine_phpcr.odm.translation.strategy.attribute.class%">
```

Comment:
I guess maybe because they require the document manager as an argument and that is set in the dependency extension.